.github/workflows/runtests.yml

```diff
@@ -1,13 +1,19 @@
 name: Run tests
 
-on: [push, pull_request]
+on:
+   push:
+     branches:
+       - master
+   pull_request:
+     branches:
+       - master
 
 jobs:
   test-julia-stable:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        julia-version: ['1.3', '1.4']
+        julia-version: ['1.4', 'nightly']
```

Comment:
what about `1.3`, we aren't dropping it as well?

Reply:
No, but I thought 1.4 would be sufficient, but I can add it back